packages/eas-cli/src/commands/channel/edit.ts

```diff
@@ -185,3 +167,25 @@ export default class ChannelEdit extends EasCommand {
     }
   }
 }
+
+async function promptForChannelAsync(): Promise<string> {
+  Log.addNewLineIfNone();
+  const { name } = await promptAsync({
+    type: 'text',
+    name: 'name',
+    message: 'Please enter the name of the channel to edit:',
+    validate: value => (value ? true : 'A channel name is required to edit a specific channel.'),
+  });
+  return name;
+}
+
+async function promptForBranchAsync(): Promise<string> {
+  Log.addNewLineIfNone();
+  const { name } = await promptAsync({
+    type: 'text',
+    name: 'name',
+    message: `To which branch should the channel link?`,
+    validate: value => (value ? true : 'branch name may not be empty.'),
```

Comment:
Please make this and https://github.com/expo/eas-cli/pull/929/files#diff-7e874262cfba75afb7089c89a7791a73ef87eadb100b81075c2edd71524e13a8R177 consistent